UI/MVUX/src/MVUX/Presentation/IMessengerSample/MessagingModel.cs

```diff
@@ -0,0 +1,41 @@
+using CommunityToolkit.Mvvm.Messaging;
+
+namespace MVUX.Presentation.IMessengerSample;
+
+public partial record MessagingModel
+{
+    protected IPeopleService PeopleService { get; }
+
+    public MessagingModel(IPeopleService peopleService, IMessenger messenger)
+    {
+        PeopleService = peopleService;
+
+        messenger.Observe(People, person => person.Id);
+    }
+
+    public IListState<Person> People =>
+        ListState
+        .Async(this, PeopleService.GetPeople)
+        .Selection(SelectedPerson);
```

Comment:
With latest version of extensions:
```suggestion
public partial record MessagingModel(IPeopleService PeopleService, IMessenger Messenger)
{
    public IListState<Person> People => ListState
        .Async(this, PeopleService.GetPeople)
        .Selection(SelectedPerson)
        .Observe(Messenger, person => person.Id);
```